.gitignore

```diff
@@ -75,3 +72,7 @@ TestResult.xcresult/
 *.xcresult/
 build.log
 *.log
+
+# Xcode workspace and scheme files (auto-generated)
+bitchat.xcodeproj/project.xcworkspace/
+bitchat.xcodeproj/xcshareddata/
```

Comment:
The files in xcshareddata/ should be added to the repo (but not the ones in xcuserdata/)